lib/torrent.js

```diff
@@ -1029,36 +1035,37 @@ export default class Torrent extends EventEmitter {
 
     this._debug('select %s-%s (priority %s)', start, end, priority)
 
-    this._selections.push({
+    this._selections.insert({
```

Comment:
SelectionItem implements `isStreamSelection` so simply pass it in the object, rather than creating a new function parameter?

Reply:
That's a good point. I removed the extra argument from `insert` and `select`, `deselect`. It's now determined by the shape of the object.